templates/standard.adoc

```diff
@@ -1,48 +1,109 @@
-= Pomegranates
+= `Vehicle`
 include::../include/config.adoc[]
-:description: A sample template for using AsciiDoc.
-:keywords: sample
+:description: The standard style template for documentation.
+:keywords: standard, template
 
-This is a sample template of how a good AsciiDoc can be structured.
+This is the standard template of how a good AsciiDoc should be structured.
 This pragraph describes the purpose of the document.
-In this sample, an imaginary API
-for producing pomegranates will be documented.
+In this example, an imaginary API for a vehicle will be documented.
 
-So without further ado, here is what we should see when we are done.
+== `Vehicle` Properties
 
-[#img-pomegranate]
-.The Expected Result
-image::pomegranate.png[Pomegranate]
+=== `make: string`
+The make of the vehicle.
 
-== Pomegranate API Methods
+=== `model: string`
+The model of the vehicle.
 
-=== `plant_tree(pos, name)`
-* `pos`: `vector`
-* `name`: `string`
+=== `year: number`
+The year of the vehicle.
 
-Plants a tree of type `name` at `pos`
-if `name` is a registered pomegranate tree.
+=== `trim: string`
+The trim of the vehicle.
 
-.Planting A Tree
+=== `color: ColorSpec`
+The color of the vehicle.
+
+== `Vehicle` Methods
+
+=== `vehicle.new(pos: vector, properties: table) -> Vehicle`
+* `pos`: The position at which the vehicle will be spanwed.
+* `properties`: A `table` of Vehicle properties (link).
+
+Creates a new instance of a `Vehicle` at the given position and returns the new object.
+
+.Creating a Vehicle
 ====
 
 [source,lua]
 ----
-plant_tree(vector.new(35, 42, 36), "regular")
+local minivan = vehicle.new(vector.new(35, 2, 36), {
+    make = "Toyota",
+    model = "Previa",
+    year = 1992,
+    trim = "LE",
+    color = "white",
+})
 ----
 
 ====
 
-=== `grow_tree(pos)`
-* `pos`: `vector`
+=== `set_properties(properties: table)`
```

Comment:
I think that it's helpful to use a placeholder object name for methods, that way you can be clear that it uses the method call syntax (`:`). I'm not sure if we should just use `object:method()` everywhere or one which is more specific to the object in question.

Reply:
Agreed, although arguably the term "methods" already conveys that.